.github/workflows/test.yaml

```diff
@@ -37,7 +37,7 @@ jobs:
         python-version: ["3.7", "3.8", "3.9", "3.10", "3.11"]
         pydantic-version: ["==1.9.0", ""] 
         dep-installer: ["dev", "pypi"]
-    continue-on-error: ${{ matrix.dep-installer == 'pypi' }}
+    continue-on-error: ${{ matrix.dep-installer == 'pypi' || matrix.python-version == '3.11' }}
```

Comment:
Looks like 3.11 is working; expect this just got re-merged back in from past versions.